io/LasWriter.cpp

```diff
@@ -524,8 +529,35 @@ void LasWriter::addGeotiffVlrs()
 /// \return  Whether the VLR was added.
 bool LasWriter::addWktVlr()
 {
+    // WKT2 and PROJJSON can be writen in PDAL VLRs
+    if (d->opts.enhancedSrsVlrs) {
+        const std::string wkt2 = m_srs.getWKT2();
+        if (!wkt2.empty()) {
+            std::vector<char> wktBytes(wkt2.begin(), wkt2.end());
+            wktBytes.resize(wktBytes.size() + 1, 0);
+            addVlr(las::TransformUserId, las::PdalWkt2RecordId, "PDAL WKT2 Record", wktBytes);
+        }
+
+        const std::string projjson = m_srs.getPROJJSON();
```

Comment:
should we always write PROJJSON? IMO it isn't that much of a cost, but I could see where people wouldn't want them written to the files and would want to control that they get WKT2 but not PROJJSON.

Reply:
Right, it isn't that much cost. If the user really want that fine control, they can create it themselves, like in the test `read_consume_order`.